charts/medusa-operator/values.yaml

```
@@ -13,8 +13,9 @@ commonLabels: {}
 replicaCount: 1
 
 image:
+  registry: docker.io
```

Missing comment for reference docs

```suggestion
  # -- Container registry containing the repository where the image resides.
  registry: docker.io
```